requirements.txt

```diff
@@ -1,7 +1,16 @@
-f90nml>=1.1.0
-appdirs>=1.4.0
-requests>=2.22.0
-pyyaml>=5.0
-gcsfs>=0.7.0
-fsspec>=0.8.0
-dacite>=1.6.0
+pip==19.2.3
+bump2version==0.5.11
+wheel==0.33.6
+watchdog==0.9.0
+flake8==3.7.8
+tox==3.14.0
+coverage==4.5.4
+Sphinx==1.8.5
+twine==1.14.0
+Click==7.0
+sphinx_rtd_theme
+pytest==5.2.2
+pytest-cov==2.8.1
+google-cloud-storage
+kubernetes==10.0.1
+black==19.10b0
```

Comment:
Please remove all `==` restrictions. `pip-compile` will automatically turn these in to direct version pins.